### ISSUE TITLE
Advance Voicetracker

### ISSUE THREAD

**Issue by olsson82** [olsson82/rivendellweb#8]
Look into on more advance voicetracker like in rivendell with multitracks.

**Comment by sjoerd276** [comment]
You could look into wavesurfer.js, there is also a multitrack example which I've used before. Maybe I could help you as well and contribute to the project. This is the link to wavesurfer: https://wavesurfer.xyz/examples/?multitrack.js

**Comment by olsson82** [comment]
> You could look into wavesurfer.js, there is also a multitrack example which I've used before. Maybe I could help you as well and contribute to the project. This is the link to wavesurfer: https://wavesurfer.xyz/examples/?multitrack.js

Yes i know about wavesurfer, it is used in recording and "audio editing" to set markers.

I have a plan to use the multitrack for this.

All help is appreciated.
